main.go

```diff
@@ -305,6 +304,13 @@ func main() {
 			SenderChecker:     senderChecker,
 			RecipientChecker:  recipientChecker(*allowedRecipients, *deniedRecipients),
 			Handler:           mailHandler,
+			MaxMessageSize:    51200000,
```

Comment:
```suggestion
			MaxMessageSize:    51200000, // Max message size in bytes
```

nit: let's add unit in comment.

Reply:
Good idea - I made these configurable, so I added it in the flag section. Let me know what you think 😄